client/src/components/More.jsx

```diff
@@ -205,6 +218,9 @@ const More = () => {
             <DialogContentText className={classes.descriptionText}>
               Enter your phone number to subscribe/unsubscribe to the occasional question/survey to answer over text.
             </DialogContentText>
+            <DialogContentText className={classes.descriptionText}>
+              Please add your +country code before entering
+            </DialogContentText>
```

Comment:
Does anyone have a better suggestion for nicely displaying in a non-engineer-y way of "Enter your phone number to match regex `/^\+[1-9]\d{1,14}$/` or `+<COUNTRY-CODE><PHONE-NUMBER>`?

Reply:
Maybe show the image on [this page](https://support.twilio.com/hc/en-us/articles/223183008-Formatting-International-Phone-Numbers) or giving a link to a website that explains it more

Reply:
Oh nice, didn't see that page - I'll add that in there 👍